performance/client/src/__tests__/primary/ExplorerAPIs/get-dataset-content.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright IBM Corporation 2020
+ */
+
+import WrkTestCase from "../../../testcase/wrk";
+import { getBasicAuthorizationHeader } from "../../../utils";
+
+class ExplorerApiDatasetContentTest extends WrkTestCase {
+  fetchZoweVersions = true;
+
+  name = "Test explorer data sets api endpoint /datasets/{ds}/content";
+  endpoint = '/api/v2/datasets/USER2.ZOWETEST.SMALL/content';
```

Comment:
We will need to find a common data set has more chance to exist in most of the systems. For example, some data set from SYS1. This test will fail on Marist system because this data set doesn't exist.

Reply:
I modified this test to instead pull the content SYS1.HELP(TIME), which should (as far as we can tell) exist on most systems.